### ISSUE TITLE
problems :D

### ISSUE THREAD

**Issue by WALKERIS** [overextended/oxmysql#163]
## Issue checklist
> Please put `x` inside of the box that matches your issue.

- [ x] I am using the latest release
- [ ] I have referenced previously reported issues
- [x ] I have referenced available documentation and cannot resolve the issue
- [x ] I'm certain this is an issue with oxymysql, and not with my resource

**Describe the bug**
when starting the server with newest esx legacy and oxmysql it doesnt start it just get stuck on starting "esx_datastore"
**Screenshots**
cant
**Code**
```lua
    -- If applicable, include the query and relevant code
```
cant
**Expected behavior**
it shuld work properly
**Additional context**

**Server details**
- FXServer artifact newest 6228
- Operating system debian9

**Database details**
- MariaDB or MySQL mariadb
- Version "Server version: 10.1.48-MariaDB-0+deb9u2 - Debian 9.13"

**Comment by darksaid98** [comment]
This is not an oxmysql issue. You said so yourself, '*... it just get stuck on starting "esx_datastore"*'.

From back when I remember that esx_datastore was a horribly architected resource that loaded an entire part of the database into memory on startup. If this is still the case, then a large table would no doubt cause startup issues. 

**TLDR**: A badly designed resource is causing this, not oxmysql.
